app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextPhoneField.kt

```diff
@@ -50,17 +76,24 @@ class EditTextPhoneField(context: Context, attrs: AttributeSet) :
 
     private fun setPhoneMask() {
         this.editText?.apply {
-            addTextChangedListener(PhoneNumberTextWatcherMask(this))
+            addTextChangedListener(PhoneNumberTextWatcherMask(mask) { setSelection(it) })
```

Comment:
### **Suggestion**
you can cover setSelection(it) with a **try/catch** if an error happened with the position to select

Reply:
done